exercises/isogram/isogram_test.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'isogram'
+
+# Test data version:
+# 306975e
+class IsogramTest < Minitest::Test
+  def test_duplicates
+    str = 'duplicates'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_eleven
+    skip
+    str = 'eleven'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_subdermatoglyphic
+    skip
+    str = 'subdermatoglyphic'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_alphabet
+    skip
+    str = 'Alphabet'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_thumbscrew_japingly
+    skip
+    str = 'thumbscrew-japingly'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_hjelmqvist_gryb_zock_pfund_wax
+    skip
+    str = 'Hjelmqvist-Gryb-Zock-Pfund-Wax'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_heizlrckstoabdmpfung
+    skip
+    str = 'Heizölrückstoßabdämpfung'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_the_quick_brown_fox
+    skip
+    str = 'the quick brown fox'
+    refute Isogram.is_isogram?(str)
+  end
+
+  def test_emily_jung_schwartzkopf
+    skip
+    str = 'Emily Jung Schwartzkopf'
+    assert Isogram.is_isogram?(str)
+  end
+
+  def test_lphant
```

Comment:
I don't like that characters are being stripped from the test names. 
(See my other comment about test names for my preferred solution.)
Why can't you use the original characters?